lib/Analysis/Membar.cpp

```diff
@@ -29,8 +29,8 @@ void MembarAnalysis::dfsOperation(Operation *operation,
         // Copy the parent info as the current info.
         RegionInfo regionInfo = *parentRegionInfo;
         for (auto &block : region.getBlocks()) {
-          assert(region.getBlocks().size() == 1 &&
-                 "Multiple blocks in a region is not supported");
+          // assert(region.getBlocks().size() == 1 &&
```

Comment:
is this commented on purpose?

Reply:
Yes. This is what the first line of the PR description is referring to. We need to comment it out or programs that use unstructured control flow will fail.